src/rred_reports/redcap/main.py

```diff
@@ -97,9 +99,9 @@ def preprocess_wide_data(cls, raw_data: pd.DataFrame, labelled_data: pd.DataFram
         return cls._rename_wide_cols_with_student_number_suffix(filtered)
 
     @staticmethod
-    def _fill_school_id_with_coalesce(raw_data, processed_extract):
+    def _fill_school_id_with_coalesce(raw_data, processed_extract, column_name):
```

Comment:
this could probably do with renaming the function name and the input data, maybe something like this (and then renaming usages of the variables and function names
```suggestion
    def _fill_school_column_with_coalesce(data_with_school: pd.DataFrame, processed_extract: pd.DataFrame, column_name: str):
```